linfa-reduction/examples/diffusion_map.rs

```diff
@@ -5,8 +5,6 @@ use ndarray_npy::write_npy;
 use ndarray_rand::rand::SeedableRng;
 use rand_isaac::Isaac64Rng;
 
-// A routine K-means task: build a synthetic dataset, fit the algorithm on it
-// and save both training data and predictions to disk.
```

Comment:
Does this comment hurt? I wouldn't mind it (but no strong opinion either).

Reply:
I believe that this example was build by modifying the `linfa-clustering` k-means algorithm example. Since this is for the diffusion map (which I think (?) is unrelated), I was under the impression this comment was just left over from that process, and could have been misleading